services/user_service/database_connector.py

```diff
@@ -0,0 +1,110 @@
+import firebase_admin
+from firebase_admin.exceptions import FirebaseError
+from firebase_admin import credentials
+from firebase_admin import db
+
+
+class DatabaseAdmin:
+
+    def __init__(self,databaseURL, credentials_path):
+        cred = credentials.Certificate(credentials_path)
+        firebase_admin.initialize_app(cred,
+        {
+            'databaseURL' : databaseURL,
+        })
+        self.__users_ref = db.reference("users")
+
+    def get_all_users(self):
+        try:
+            if self.__users_ref.get() is None:
+                return list()
+            return self.__users_ref.get()
+        except FirebaseError as e:
+            print("###Connection lost. EROR received : " + str(e))
+            return list()
+    
+
+    def get_user_by_id(self, id):
+        try:
+            if self.__users_ref.child(id).get() is None:
+                return list()
+            return self.__users_ref.child(id).get()
+        except ValueError as e:
+            print("###EROR received : " + str(e))
+        except FirebaseError as e:
+            print("###Connection lost. EROR received : " + str(e))
```

Comment:
We need to consider exiting the app if there's an error, so we can restart the service and it continue working correctly.

Reply:
Yes, but not at this point. If we do not have an internet connection, then we will need to restart all services I believe, since the will be deployed on the same cloud.